Automattic/newspack-plugin
# SVG Dashboard Icons

### All Submissions:






### Changes proposed in this Pull Request:



Replaces dashboard .pngs with .svgs.

### How to test the changes in this Pull Request:

1. `npm ci && npm run clean && npm run build:webpack`
2. Verify dashboard visuals are essentially unchanged.
3. Verify all dashboard icons are .svgs.

### Other information:
